api/tests/test_cloud_record_summary.py

```diff
@@ -90,17 +99,21 @@ def test_cloud_record_summary_get_401(self):
         """Test an unauthenticated GET request."""
         test_client = Client()
         # Test without the HTTP_AUTHORIZATION header
-        response = test_client.get('/api/v1/cloud/record/summary?'
-                                   'group=TestGroup&'
-                                   'from=20000101&to=20191231')
+        get_url = ''.join((reverse('CloudRecordSummaryView'),
```

Comment:
Repetition within the same function. You could reuse the `url` as the argument to the `.get()`